components/FoodCard/FoodCard.js

```diff
@@ -0,0 +1,37 @@
+import styled from "styled-components";
+
+export default function MenuCard({ food }) {
+  return (
+    <>
+      <StyledFoodCard>
+        <StyledHeading>{food.name}</StyledHeading>
+        <StyledParagraph>{food.description}</StyledParagraph>
+        <Price>{food.price}€</Price>
+      </StyledFoodCard>
+    </>
+  );
+}
+
+const StyledFoodCard = styled.article`
+  width: 100%;
+  margin: 0.5rem;
+  padding: 0.3rem;
+
+  display: flex;
+  flex-direction: column;
+
+  border-radius: 10px;
+`;
+
+const StyledHeading = styled.p`
```

Comment:
`h2` or similar?

Reply:
I just noticed that, might be the name  "FoodCard" led to a misunderstanding. This component  ist only for one dish. all the dishes together will be the menu. So I didn't use H2. or, should I name it as "FoodListItem"?